src/translations/screens/Assistant.ts

```diff
@@ -54,7 +54,7 @@ const AssistantTexts = {
     },
   },
   searchState: {
-    searching: _('Laster søkeresultater', 'Loading search results'),
+    searching: _('Laster søkeresultater...', 'Loading search results...'),
```

Comment:
Det er ikke lagt inn avsluttende ellipse på tilsvarende tekst "Søker etter flere reiseforslag" i `results.fetchingMore`. Det børe være likt de to stedene. 

Skissene i Figma har heller ingen avsluttende ellipse. Jeg ville droppet dette nå, men hvis du brenner for ellipsen så ta en rask avklaring med Hilde/Simen 😄

Reply:
Tenkte at "..." (ellipse?) kan fungere som en indikasjon på at det er noe som pågår selv om spinner ikke er synlig. Ellers føltes den veldig statisk syns jeg. 

"Søker etter flere reiseforslag" har alltid synlig spinner, så kanskje ikke nødvendig der.

Men gir gjerne design-hatten videre til noen som faktisk er flink på sånt 😅 Tanker @strombraaten eller @hildeor?

Reply:
Angående "..."/ellipse så tenker jeg ellipse er riktig å bruke her. Og da bruk ellipse-tegnet og ikke tre punktum (https://www.korrekturavdelingen.no/ellipse-tre-prikker.htm).

Er ikke det alltid synlig (native) spinner ovenfor teksten "Laster søkeresultater"?

Reply:
Aah, det du mener. Visste ikke at tegnet ble kalt ellipse.

```suggestion
    searching: _('Laster søkeresultater…', 'Loading search results…'),
```

Ikke alltid den er synlig nei, kun når man scroller oppover på iOS.

https://user-images.githubusercontent.com/1774972/160118681-0b2fc5a1-22db-438a-a1a9-4c54d6a8f9e6.mp4